<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROJETO CONTA POUPANÃ‡A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOME DO CLIENTE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AGENCIA: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NUMERO DA CONTA : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SALDO :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomeDoCliente">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAgencia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumeroDaConta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSaldo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValor">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VALOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCriarConta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar Conta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSacar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SACAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDepositar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>DEPOSITAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
